src/FileLoader.java

```diff
@@ -0,0 +1,100 @@
+
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileLoader {
+
+
+    public static List<Orders> loadFile(String fileName) {
+        ArrayList<Orders> listOfOrders = new ArrayList();
+        Path pathToFile = Paths.get(fileName);
+        try {
+
+            BufferedReader br = Files.newBufferedReader(pathToFile);
+
+            String line = br.readLine();
+
+
+            while (line != null) {
+
+                String[] attributes = line.split(",");
+
+                if (checkOrders(attributes) == true) {
+                    Orders order = createObject(attributes);
+
+                    if (order != null) {
+                        listOfOrders.add(order);
+                    }
+                }
+
+                line = br.readLine();
+            }
+
+
+        } catch (IOException e) {
+            System.out.println("Format error occured.");
+            e.printStackTrace();
+        }
+        return listOfOrders;
+    }
+
+
+    public static Orders createObject(String[] metaData) {
+        try {
+            int client_ID = Integer.parseInt(metaData[0]);
+            int request_ID = Integer.parseInt(metaData[1]);
+            String name = metaData[2];
+            int quantity = Integer.parseInt(metaData[3]);
+            double price = Double.parseDouble(metaData[4]);
+            return new Orders(client_ID, request_ID, name, quantity, price);
+        } catch (NumberFormatException e) {
```

Comment:
Why do you still have exception here if you earlier validated data to be correct?